apis/network/v1beta1/vpc_types.go

```diff
@@ -31,43 +30,58 @@ type VPCParameters struct {
 	CIDRBlock string `json:"cidrBlock"`
 
 	// A boolean flag to enable/disable DNS support in the VPC
-	EnableDNSSupport bool `json:"enableDnsSupport,omitempty"`
-
-	// A boolean flag to enable/disable DNS hostnames in the VPC
-	EnableDNSHostNames bool `json:"enableDnsHostNames,omitempty"`
+	// +optional
+	EnableDNSSupport *bool `json:"enableDnsSupport,omitempty"`
 
 	// Tags are used as identification helpers between AWS resources.
+	// +optional
 	Tags []Tag `json:"tags,omitempty"`
+
+	// Indicates whether the instances launched in the VPC get DNS hostnames.
+	// +optional
+	EnableDNSHostNames *bool `json:"enableDnsHostNames,omitempty"`
+
+	// The allowed tenancy of instances launched into the VPC.
+	// +optional
+	InstanceTenancy *string `json:"instanceTenancy,omitempty"`
 }
 
 // A VPCSpec defines the desired state of a VPC.
 type VPCSpec struct {
 	runtimev1alpha1.ResourceSpec `json:",inline"`
-	VPCParameters                `json:",inline"`
+	ForProvider                  VPCParameters `json:"forProvider"`
 }
 
-// VPCExternalStatus keeps the state for the external resource
-type VPCExternalStatus struct {
+// VPCObservation keeps the state for the external resource
+type VPCObservation struct {
+	// Indicates whether the VPC is the default VPC.
+	IsDefault bool `json:"isDefault,omitempty"`
+
+	// The ID of the AWS account that owns the VPC.
+	OwnerID string `json:"ownerID,omitempty"`
+
 	// VPCState is the current state of the VPC.
 	// +kubebuilder:validation:Enum=pending;available
```

Comment:
```suggestion
```

Since this is not a user input, i.e. under status, we can delete the validation.